spec/spec_helper.rb

```diff
@@ -5,8 +5,8 @@
   minimum_coverage 97
 end
 
-Dir['./spec/support/**/*'].each do |f|
-  require f.sub(%r{\./spec/}, '')
+Dir['./spec/support/**/*.rb'].each do |f|
+  require f.sub(%r|\./spec/|, '')
```

Comment:
Revert this unless needed please :)

Reply:
I can't:
```
LoadError:
  cannot load such file -- support/shared_examples
```

Reply:
Reverted the `%r{}` -> `%r||` change. It triggers RuboCop locally, but we don't check `spec/` on CI.